packages/ui/src/components/MultixIdenticon.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react'
+import Tooltip from '@mui/material/Tooltip'
+import { ICON_SIZE, ICON_THEME } from '../constants'
+import Identicon from '@polkadot/react-identicon'
+import { IconButton } from '@mui/material'
+import { IconTheme } from '@polkadot/react-identicon/types'
+
+const DEFAULT_PLACEMENT = 'top'
+const DEFAULT_TITLE = 'Copied'
```

Comment:
Maybe it would be clearer to our users what is actually copied
```suggestion
const DEFAULT_TITLE = 'Address copied!'
```

Reply:
Hey @Tbaut,  Is it always ADDRESS what we copy throughout the app?

Reply:
Hey @Tbaut  Also Is there any way i can click on CREATE button here. I am sure we are using rococo testing environment here to send ROC etc.

![image](https://github.com/ChainSafe/Multix/assets/61174028/7e3b3f8e-d019-436a-8f90-c4bca14f2278)

Reply:
The error says you don't have enough ROC on the Signing account. Get some ROC using the faucet: https://matrix.to/#/#rococo-faucet:matrix.org

> Hey @Tbaut, Is it always ADDRESS what we copy throughout the app?

The copy action is from the `Identicon` component that we take from a library, so we don't handle the copy per se, and yet it'll always be the address copied in this case

Reply:
> The error says you don't have enough ROC on the Signing account. Get some ROC using the faucet: https://matrix.to/#/#rococo-faucet:matrix.org


@Tbaut  it is showing choose app to continue. Can i choose any one of them?

![image](https://github.com/ChainSafe/Multix/assets/61174028/0c1a0678-f70a-480e-9573-af2cb48f702c)

Reply:
I can answer this one @dj-pundir. Yep, you should be able to use any of them. Element seems to be the most popular.